components/docs-chef-io/content/automate/major_upgrade_4.x.md

```diff
@@ -243,76 +334,114 @@ sudo ./chef-automate config show
 
 **Post Upgrade**
 
 1. Check the upgrade status of Chef Automate, run the following command:
-
 ```sh
 sudo chef-automate upgrade status
 ```
+This should return: Automate is up-to-date with airgap bundle `4.x.y` version
 
 2. Turn off the maintenance mode.
-
 ```sh
 sudo chef-automate maintenance off
 ```
+This should return:
+  
+  ```result
+  Updating deployment configuration 
+  Applying deployment configuration
+  ```
 
-3. All [relevant configuration fields](https://docs.chef.io/automate/opensearch/) of the Elasticsearch should be copied into the OpenSearch configuration.
+3. Migrate your data from *ElasticSearch 6.8* to *OpenSearch 1.2.4*:
+```sh
+sudo chef-automate post-major-upgrade migrate --data=es
+```
+```shell
+It will start the migration immediately after check.
+Press y to agree, n to disagree? [y/n]: y
 
-For Example:
+HAB Root Path /hab/pkgs/chef/deployment-service/0.1.0/20220609123606
 
-If Elasticsearch configuration was:
 
-```bash
-[elasticsearch]
-  [elasticsearch.v1]
-    [elasticsearch.v1.sys]
-      [elasticsearch.v1.sys.cluster]
-        max_shards_per_node = 6000
-      [elasticsearch.v1.sys.indices]
-        [elasticsearch.v1.sys.indices.breaker]
-          total_limit = "95%"
-      [elasticsearch.v1.sys.runtime]
-        max_open_files = "65536"
-        max_locked_memory = "unlimited"
-        heapsize = "8g" # This should be the 50% of RAM```
-```
+----------------------------------------------
+Chef-automate stop 
+----------------------------------------------
 
-Then add in OpenSearch configuration as:
+Chef Automate Stopped
 
-```bash
-[opensearch]
-  [opensearch.v1]
-    [opensearch.v1.sys]
-      [opensearch.v1.sys.cluster]
-        max_shards_per_node = 6000 # Refer the value from ElasticSearch Config, If this value is not there in elastic search config, then do not patch in openseaarch.
-      [opensearch.v1.sys.indices]
-        [opensearch.v1.sys.indices.breaker]
-          total_limit = "95%"
-      [opensearch.v1.sys.runtime]
-        max_open_files = "65536"
-        max_locked_memory = "unlimited"
-        heapsize = "8g" # This should be the 50% of RAM
+----------------------------------------------
+migration from es to os 
+----------------------------------------------
 
-```
+Checking for es_upgrade
 
-Apply this using `config patch` command.
+Done with Migration 
+ Please wait for some time to reindex the data
 
-4. Migrate your data from *ElasticSearch 6.8* to *OpenSearch 1.2.4*:
+----------------------------------------------
+Chef-automate start 
+----------------------------------------------
 
-```sh
-sudo chef-automate post-major-upgrade migrate --data=es
-```
+Starting Chef Automate
 
-5. Verify whether all services are running:
+----------------------------------------------
+Chef-automate status 
+----------------------------------------------
+```
+When migration is successful the default config will be shown
```

Comment:
are you sure about this? I don't see any config with successful migration

Reply:
Yes with the latest changes made the config will be displayed